CgfConverter/CryEngineCore/Chunks/ChunkNode.cs

```diff
@@ -17,13 +17,14 @@ public abstract class ChunkNode : Chunk          // cccc000b:   Node
     public bool IsGroupHead { get; internal set; }
     public bool IsGroupMember { get; internal set; }
     public Matrix4x4 Transform { get; internal set; }
-    public Vector3 Pos { get; internal set; }
-    public Quaternion Rot { get; internal set; }
-    public Vector3 Scale { get; internal set; }
-    public int PosCtrlID { get; internal set; }  // Obsolete
-    public int RotCtrlID { get; internal set; }  // Obsolete
-    public int SclCtrlID { get; internal set; }  // Obsolete
+    public Vector3 Pos { get; internal set; }       // Obsolete
```

Comment:
Good catch.  I saw this in the Lumberyard source, but never transferred it back over to here.

Reply:
You know what, I absolutely do not recall making changes to this part. Will check again for this one.

Reply:
I was looking in my current animations branch (`issue-145/animations`) and see the change there, as well as a couple of other places.  I think you may have pulled in that branch, which is good because it had a few other things I wanted to bring in. :)